app/logic/emailHandler.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, request, render_template, redirect, url_for
+import yaml, os
+from flask_mail import Mail, Message
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.program import Program
+from app.models.eventParticipant import EventParticipant
+from app.models.emailTemplate import EmailTemplate
+from app import app
+import sys
+import webbrowser
+import time
+from pathlib import Path
+
+def getInterestedEmails(listOfPrograms):
+
+    """ Gets emails of students interested in a program. """
+    emails = []
+
+    volunteersToEmail = User.select(User.email).join(Interest).join(Program, on=(Program.id == Interest.program)).where(Program.id.in_([p.id for p in listOfPrograms]))

```

Comment:
This line needs to be split up

Reply:
`emailHandler.py` is from the original PR. We didn't change this one because we didn't know if we could delete it. 
If the class defined in `emailHanlder2.py` is doing everything we need it to do, then we can delete this one. 

I believe it does, so we might as well this file altogether.